benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -25,13 +25,14 @@ static constexpr size_t SKYBOX_UNIFORM_BUFFER_REGISTER = 0;
 static constexpr size_t SKYBOX_SAMPLED_IMAGE_REGISTER  = 1;
 static constexpr size_t SKYBOX_SAMPLER_REGISTER        = 2;
 
-static constexpr size_t SPHERE_UNIFORM_BUFFER_REGISTER                = 0;
-static constexpr size_t SPHERE_ALBEDO_SAMPLED_IMAGE_REGISTER          = 1;
-static constexpr size_t SPHERE_ALBEDO_SAMPLER_REGISTER                = 2;
-static constexpr size_t SPHERE_NORMAL_SAMPLED_IMAGE_REGISTER          = 3;
-static constexpr size_t SPHERE_NORMAL_SAMPLER_REGISTER                = 4;
-static constexpr size_t SPHERE_METAL_ROUGHNESS_SAMPLED_IMAGE_REGISTER = 5;
-static constexpr size_t SPHERE_METAL_ROUGHNESS_SAMPLER_REGISTER       = 6;
+static constexpr size_t SPHERE_COLOR_BUFFER_REGISTER                  = 0;
+static constexpr size_t SPHERE_UNIFORM_BUFFER_REGISTER                = 1;
```

Comment:
the be consistent with the naming, probably better with SPHERE_SCENEDATA_UNIFORM_BUFFER_REGISTER

Reply:
Done.